Dockerfile

```diff
@@ -35,9 +35,14 @@ RUN apt-get update && \
     pip3 install ccsyspath
 
 # Build dependencies
-RUN git clone --depth=1 --branch master https://github.com/lifting-bits/remill.git && \
-    cd remill && \
-    ./scripts/build.sh --llvm-version ${LLVM_VERSION} --prefix ${LIBRARIES} --download-dir /tmp
+RUN git clone https://github.com/lifting-bits/remill.git
+RUN if [ "$LLVM_VERSION" -gt 11 ]; then \
+    cd remill && git checkout -b release_93aba7c 93aba7c && \
+    ./scripts/build.sh --llvm-version ${LLVM_VERSION} --prefix ${LIBRARIES} --download-dir /tmp; \
+    else \
+    cd remill && git checkout -b all_llvm 9006baf7db && \
+    ./scripts/build.sh --llvm-version ${LLVM_VERSION} --prefix ${LIBRARIES} --download-dir /tmp; \
+    fi
```

Comment:
This should follow what we have in the GitHub Actions CI file:

* Only test/support up to LLVM 11, so remove the conditional check for checking out different LLVM versions
* Change the remill commit to the one used in CI on [this line](https://github.com/lifting-bits/mcsema/blob/b7492e2ffb9f2fa64b5ef269753ba1d134bdf2f2/.github/workflows/vcpkg_ci.yml#L41)